test/pipeline_test.bzl

```diff
@@ -38,6 +55,18 @@ def pipeline_tests(all_paths, test_name_prefix, filter = "*", extra_args = [], t
                 "{}/{}*".format(path, prefix),
             ]) + ["test_corpus_sharded"],
             size = "small",
-            tags = tags,
+            tags = tags + extra_tags,
+        )
+
+    if enabled_tests:
+        native.test_suite(
+            name = suite_name,
+            tests = enabled_tests,
+        )
+
+    if disabled_tests:
```

Comment:
why not always?

Reply:
That's a fair point. I'll remove the logic to conditionally emit the test suites.